testsuite/libffi.closures/huge_struct.c

```diff
@@ -5,7 +5,7 @@
 	Originator:		Blake Chaffin	6/18/2007
 */
 
-/* { dg-do run { xfail strongarm*-*-* xscale*-*-* } } */
+/* { dg-do run { xfail strongarm*-*-* xscale*-*-* wasm32*-*-* } } */
```

Comment:
What about `libffi.closures`tests `huge_struct.c` and `closure_loc_fn_0.c`? These should fail too.

Reply:
Commit https://github.com/hoodmane/libffi-emscripten/pull/4/commits/254af96bc0231077bc2947be5090b6b343ed44ab changed those tests to `xfail` ones.